.github/workflows/schedule.yaml

```diff
@@ -11,5 +11,3 @@ on:
 jobs:
   test:
     uses: ./.github/workflows/test.yaml
-    with:
-      checkout-ref: ${{ matrix.checkout-ref }}
```

Comment:
I was trying to minimise changes not necessary to prevent the build failing. This just popped up as an error in my IDE since it isn't run as part of a matrix. I've done some additional tidy-up of the build definitions.